website/content/userguide/rsync-backups.markdown

```diff
@@ -77,8 +77,12 @@ dont_namespace = true
 EOT
 ```
 
+(Note that `scan2drive.lan` is the name of my gokrazy instance. I added the IP of the instance and defined a name in `/etc/hosts`, i.e `192.168.10.2 scan2drive.lan`)
+
 Finally, create the authorized keys file that determines who can access the rsync daemon:
 
+(i.e `ssh-keygen -t ed25519 -C "michael@midna.com"`, save in `~/.ssh/id_ed25519_scan2drivebackup` and cat the `.pub` key to `gokr-rsyncd.authorized_keys`)
```

Comment:
I don’t think this line is good to add — it repeats what the command below it already says.

You don’t need to generate an ed25519 key specifically (it was the default on my machine), and the comment was likewise auto-generated. 

Creating an authorized keys file should be familiar to SSH users. If you want, we can add a link to a tutorial (or documentation) regarding SSH public keys and authorized_keys files, but I don’t want to spell it out in terms of “run ssh-keygen with these options”

Reply:
Ok sure I understand!

Reply:
@stapelberg if you want just approve my other commit about `waiw`  and lets not make changes on the docs

Reply:
I don’t know how to approve only a single commit. Can you change the PR to only contain one commit please?

Reply:
@stapelberg there you go!